cityofaustin/atd-vz-data
# Set VZV dataset endpoints based on environment

## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/18689
- https://github.com/cityofaustin/atd-data-tech/issues/18688

## Testing

1. start your local VZV
2. Make sure the VZV looks ok. At the moment there's not much data in the staging dataset.
3. I'm not sure how to test the prod changes?


---
#### Ship list
